homeassistant/components/dyson/fan.py

```diff
@@ -155,6 +176,14 @@ def service_handle(service):
     hass.services.register(
         DYSON_DOMAIN, SERVICE_SET_AUTO_MODE, service_handle, schema=SET_AUTO_MODE_SCHEMA
     )
+
+    hass.services.register(
```

Comment:
Please register the services using our entity service helper:
https://developers.home-assistant.io/docs/dev_101_services#entity-services

Note how to pass the discovery schema as a dict with validators.